### PR TITLE
First step for Language Compatibility

### PR DESCRIPTION
**Pull request by Terasol**
This is a first stab at creating language compatibility based on Emerald.
From what i could find, all languages other then Japanese use the same memory layout.
Only the rom addresses need to be translated.

This currently allows the modes spin and fishing to work.
for starters more of the rom addresses would need to be translated and more complicated "gRngValue"

### CODE REVIEW

File: pokebot.py [hunk 1]
```diff
@@ -40,16 +40,13 @@ def _exit() -> NoReturn:
     try:
         if config_general['bot_mode'] != 'manual':
             if GetGameState() == GameState.BATTLE:
-                # Search for the text "What will (Pokémon) do?" in `gDisplayedStringBattle`
-                b_What = EncodeString('What')  # TODO English only
-
-                while ReadSymbol('gDisplayedStringBattle', size=4) != b_What:
+                while GetTask('TASK_HANDLEMONANIMATION').get('data', b'0x00000000')[8] != 0: #define tBattlerId data[4]
```
Comment (40Cakes) [inline pokebot.py:43]: This doesn't work with battle scene set to off 😅

Using this instead, seems to work in all games:
```
while GetTask('TASK_PLAYCRYWHENRELEASEDFROMBALL') == {} and GetTask('SUB_8046AD0') == {}:
    PressButton(['B'])
```

Comment (40Cakes) [reply pokebot.py:43]: Nvm, this frequently gets stuck at high speeds 😑

Comment (Terasol) [reply pokebot.py:43]: at least for Emerald it works... for all languages..

Comment (40Cakes) [reply pokebot.py:43]: Battle scene: off disables animations, so `TASK_HANDLEMONANIMATION` is never called.

Comment (Terasol) [reply pokebot.py:43]: ye i am with you on that.. was just looking at the one you proposed

Comment (40Cakes) [reply pokebot.py:43]: Ah right, yeah I'm retrying it - I think I was getting stuck somewhere else (I'm just testing a feature to walk back to home tile if the bot wanders off while spinning to win)

**Pull request merged**